app/src/server/api/routers/profile.ts

```diff
@@ -746,6 +748,19 @@ export const profileRouter = createTRPCRouter({
         limit: 5,
       });
     }),
+  getUserDailyPveBattleCount: protectedProcedure
```

Comment:
Since we are using this on all public profiles, and likely want it visible to everyone, I'm thinking we should add it directly the `getPublicUser` endpoint. I believe it can be added directly to the `with` statement on the `getPublicUser` query.

Reply:
There's currently no relationship to battleHistory from userData at least that I can include on With.
Also if I were to add it then all battle history would be pulled making the query quite heavy. Although its a new query, I did make it very light and targeted.

I changed the PvE fights to be publically viewable though.